aws/database/opensearch/client/elasticsearch_client.py

```diff
@@ -6,6 +6,30 @@
 # Suppress warnings about insecure connections - this is because we're connecting to localhost and not using SSL
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
+
+def create_client(index: str | None, port: int = 3333) -> Elasticsearch:
+    """Create a connection to a local Elasticsearch instance and check that it's working"""
+    es_instance = Elasticsearch(
+        hosts=[{"host": "localhost", "port": port}],
+        use_ssl=True,
+        verify_certs=False,
+        index=index,
+    )
+
+    try:
+        es_instance.info()
+    except elasticsearch.exceptions.ConnectionError:
+        print(f"Could not connect to ES at localhost:{port} - are you port forwarding?")
+
+    if index is not None and not es_instance.indices.exists(index):
+        valid_indices = es_instance.indices.get_alias(index="*").keys()
+        raise ValueError(
+            f"Index {index} does not exist. Valid indices are {valid_indices}"
+        )
+
+    return es_instance
```

Comment:
I split this logic out so that this can be used without the LocalElasticsearchClient